Dockerfile

```diff
@@ -3,7 +3,8 @@ FROM golang:1.15.2 as builder
 ENV GOROOT=/usr/local/go
 
 # Install deps for filecoin-project/filecoin-ffi
-RUN apt-get update && apt-get install -y jq mesa-opencl-icd ocl-icd-opencl-dev hwloc libhwloc-dev
+RUN apt-get update
```

Comment:
I don't think we should split these lines, otherwise the update will be cached separately to the install. This caused a problem for me when adding hwloc since some of the cached apt repositories had changed and threw errors.

Reply:
Good callout. 🤝